packages/gravity-ui-web/filesizehistory.html

```diff
@@ -0,0 +1,95 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="https://static.buildit.digital/gravity-ui-web/latest/gravity.css">
+    <script type="text/javascript" src="develop-file-size-history.js"></script>
+    <script type="text/javascript" src="master-file-size-history.js"></script>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
```

Comment:
Would probably be best practice to throw this JS into a separate file.

Reply:
While I agree, I'd say this is low priority for now.

I think it would be nice to spend a bit of time polishing the layout and design of this page and the versions page. If we did that, it could also be an opportunity to tidy up JS.

But, I'd suggest that's done as a future enhancement and doens't block this PR.